frontend/public/components/factory/list-page.jsx

```diff
@@ -31,21 +31,23 @@ export const TextFilter = (props) => {
   const {
     label,
     className,
-    placeholder = `Filter ${label}...`,
+    placeholder,
     autoFocus = false,
     parentClassName,
     ...otherInputProps
   } = props;
   const { ref } = useDocumentListener();
+  const { t } = useTranslation();
+  const placeholderText = placeholder || t('public~Filter {{label}}...', { label });
```

Comment:
I think this is probably OK, but note that it's slightly different. You can no longer clear a placeholder by passing `''`. If we want the same behavior as before, it needs to be

```suggestion
  const placeholderText = placeholder ?? t('public~Filter {{label}}...', { label });
```

Reply:
Oh, I didn't realize that was a thing for this component. Thanks for this.